en/middleware/micrortps.md

```diff
@@ -44,12 +46,14 @@ set(config_rtps_send_topics
    )
 
 set(config_rtps_receive_topics
-   vehicle_command
+   sensor_baro
    # Add new topic...
    )
 ```
 
-> **Note** There is another way to specify which messages are intended to be used calling **generate_microRTPS_bridge.py** using `-s` (send) and `-r` (receive) flags. Please see [Manual Generation of the Code](../middleware/micrortps_manual_code_generation.md).
+> **Caution** At time of writing (August 2017), only the small set of uORB topics listed above are included in our cmake files: **posix_sitl_default.cmake**, **nuttx_px4fmu-v4_default.cmake**, **posix_sdflight_default.cmake**. It is likely you will need to edit your *cmake* file and add additional uORB topics. In future we hope to define a larger standard set. 
```

Comment:
No these aren't really inherited except for posix_sitl_default by shared by posix_sitl_{broadcast, ekf2, inav, lpe, shell, test} and sdflight shared by eagle and excelsior. I suppose that sounds like the opposite of what I said, but the posix_sitl_default setup is more of a hack until there's a better init system.